tracer/src/Datadog.Trace/Debugger/Symbols/SymbolsUploader.cs

```diff
@@ -136,9 +136,9 @@ private void UnRegisterToAssemblyLoadEvent()
             AppDomain.CurrentDomain.AssemblyLoad -= CurrentDomain_AssemblyLoad;
         }
 
-        private async void CurrentDomain_AssemblyLoad(object? sender, AssemblyLoadEventArgs args)
+        private void CurrentDomain_AssemblyLoad(object? sender, AssemblyLoadEventArgs args)
         {
-            await ProcessItemAsync(args.LoadedAssembly).ConfigureAwait(false);
+            _ = ProcessItemAsync(args.LoadedAssembly);
```

Comment:
This task is unobserved right, and given `ProcessItemAsync` may throw (e.g. `WaitAsync()` could throw Cancellation exception), could that not be an issue? It's a niche edgecase, but as I assume we just need another try-catch in `ProcessItemAsync`, is that worth doing?

Reply:
Unobserved task exceptions are not an issue since .NET 4.5, but I guess it can't hurt to log the exception.

Reply:
> Unobserved task exceptions are not an issue since .NET 4.5

For some reason I thought they were still a thing on .NET FX, in which case, meh